lib/bemhtml/index.js

```diff
@@ -116,6 +116,7 @@ BEMHTML.prototype.render = function render(context,
       classValue += ' ' + (typeof cls === 'string' ?
                     utils.attrEscape(cls).trim() : cls);
   } else {
+    /* istanbul ignore else */
```

Comment:
Here `cls` — value from `cls()` mode or `cls` field from BEMJSON.

Again, I just instructed Istambul to omit `else` branch for `if` in 120 line.